docs/user/howto/manage-applications.md

```diff
@@ -460,33 +460,56 @@ juju bind <application> new-default endpoint-1=space-1
 
 > See more: {ref}`command-juju-bind`
 
+
+(observe-an-application)=
+## Observe an application
+
+To observe an application, on a separate Kubernetes model deploy the Canonical Observability Stack, then set up all the necessary cross-model relations. Alternatively, you can deploy only the charms pieces in the stack that you need immediately.
+
+
+> See more: [Charmhub | Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack)
+
+
+(migrate-an-application)=
+## Migrate an application
+
+To migrate an application from one controller to another, migrate the model that it has been deployed to.
+
+> See more: {ref}`migrate-a-workload-model-to-another-controller`
+
 (upgrade-an-application)=
 ## Upgrade an application
 
 To upgrade an application, update its charm. 
 
 > See more: {ref}`update-a-charm`
 
-
+(remove-an-application)=
 ## Remove an application
 > See also: {ref}`removing-things`
 
 
 To remove an application, run the `remove-application` command followed by the name of the application. For example:
 
+```{caution}
+Removing an application which has relations with another application will terminate that relation. This may adversely affect the other application. 
+```
+
+
 ```text
 juju remove-application apache2
 ```
 
-It can take a while for the application to be completely removed but if juju status reveals that the application is listed as ‘dying’, but also reports an error state, then the removed application will not go away. See the section below for how to manage applications stuck in a dying state.
-
 This will remove all of the application's units. All associated resources will also be removed providing they are not hosting containers or another application's units.
 
+
+```{note}
+It can take a while for the application to be completely removed. However, it may also happen that the application goes into an error state. In that case try `juju resolved --no-retry <unit>`; e.g., `juju resolved --no-retry mongodb-k8s/0`.
```

Comment:
question: should the note mention that all the related pieces being removed because the application is, are listed when you run the command.

Reply:
Good idea. I'll include that, thanks!